src/Property/Selector.php

```diff
@@ -77,7 +77,7 @@ class Selector
      */
     public static function isValid($sSelector)
     {
-        return preg_match(static::SELECTOR_VALIDATION_RX, $sSelector);
+        return preg_match(static::SELECTOR_VALIDATION_RX, $sSelector) && strpos($sSelector, ': ') === false;
```

Comment:
I guess this check is probably too simplistic since it will fail many valid selectors, e.g.:

* `a[href~=": "]`
* `#some_id_that_ends_in_a\: > a`

Reply:
Thanks! These should be fixed now :)